heron/scheduler-core/src/java/com/twitter/heron/scheduler/UpdateTopologyManager.java

```diff
@@ -177,7 +167,7 @@ private void updateTopology(final PackingPlans.PackingPlan existingProtoPackingP
       // delete the packing plan. Instead we could message tmaster to invalidate the physical plan
       // and/or possibly even update the packing plan directly
       SysUtils.sleep(Duration.ofSeconds(10));
-      reactivateTopology(stateManager, updatedTopology, removableContainerCount);
+      reactivateTopology(stateManager, topology, removableContainerCount);
```

Comment:
Good to have a comment here saying:
`Will throw exceptions internally if failed to xxxx`

Reply:
done